src/random.cpp

```diff
@@ -191,6 +191,7 @@ void GetDevURandom(unsigned char *ent32)
     do {
         ssize_t n = read(f, ent32 + have, NUM_OS_RANDOM_BYTES - have);
         if (n <= 0 || n + have > NUM_OS_RANDOM_BYTES) {
+            close(f);
```

Comment:
I'm so used to try to bring everything down brick by brick, I can't remember ever resorting to abort, or even exit :worried: